apps/web/src/pages/JobPosterTemplates/JobPosterTemplatePage/JobPosterTemplatePage.tsx

```diff
@@ -209,8 +209,4 @@ const JobPosterTemplatePage = () => {
   );
 };
 
-export const Component = () => <JobPosterTemplatePage />;
-
-Component.displayName = "JobPosterTemplatePage";
```

Comment:
Why remove Component.displayName in this component but not elsewhere?

Reply:
This is exported with a name. The `displayName` is only really there so that when using react dev tools, you don't see a bunch of `Component` in the tree.